Cargo.toml

```diff
@@ -166,7 +166,9 @@ solang-parser = "=0.3.3"
 # no default features to avoid c-kzg
 revm = { version = "13.0.0", default-features = false }
 revm-primitives = { version = "8.0.0", default-features = false }
-revm-inspectors = { version = "0.5", features = ["serde"] }
+# revm-inspectors = { version = "0.5", features = ["serde"] }
+# TODO
+revm-inspectors = { path = "/Users/sohamzemse/Workspace/opensource/external/paradigm/revm-inspectors", features = ["serde"] }
```

Comment:
could you please turn this into a patch to https://github.com/paradigmxyz/revm-inspectors so that CI works and make this PR into a draft so we don't merge it accidentally?